services/ingestion/domains/matches/dtos.py

```
@@ -0,0 +1,30 @@
+# services/ingestion/domains/matches/dtos.py
+from typing import List
```

Ruff (0.15.2)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)